sdk/python/tests/test_connection.py

```diff
@@ -0,0 +1,13 @@
+from pvsite_datamodel import DatabaseConnection
+
+from pvsite_datamodel.sqlmodels import (
+    SiteSQL,
+)
+
+class TestDatabaseConnection:
+    """Tests for the DatabaseConnection class."""
+
+    def test_connection(self, engine, sites):
+        dbcon = DatabaseConnection(engine.url, echo=False)
+        with dbcon.get_session() as session:
+            session.query(SiteSQL).first()
```

Comment:
thanks for adding this

Reply:
Yeah I actually removed it by accident: it was in the `test_write.py` file.